provider/testdata/data/cldr/cldr-cal-japanese-full/main/sr-Cyrl/ca-japanese.json

```diff
@@ -0,0 +1,1248 @@
+{
+  "main": {
+    "sr-Cyrl": {
+      "identity": {
+        "version": {
+          "_cldrVersion": "40"
+        },
+        "language": "sr",
+        "script": "Cyrl"
+      },
+      "dates": {
+        "calendars": {
+          "japanese": {
```

Comment:
`question`: this whole subtree seems to be likely 1-1 match to gregorian. To save on the payload, could we keep a single copy of it per locale, and only store overrides in each calendar?

Reply:
I think this applies more broadly to a lot of variant-data; perhaps file a followup? It's kinda orthogonal to this PR since this is all stuff that already existed.

Reply:
Note that this file is a `cldr` file, not an actual ICU4X JSON file.

Reply:
Yep, though his comment does apply to the ICU4X side too